atlp-rwanda/Phantom-frontend-pirates
# Registering drivers&operators

# Registering employees

## Overview
In this feature  I have completed Registration of employees and their management  where an admin can 

-Register an employee and assign him/her a role
-Update his/her role
-Delete the employee

## Testing 

To test this follow these steps 

1. Pull this branch
2. run `npm install` to install all dependencies in `package.json`
3.  run `npm run dev` to start a server 
5. Your browser will be opened and navigate to `/login` 
6. Login as an admin and go to the sidebar to click on **Employees** to get a list of all employees
7. start by registering a new employee and an email will be sent to that email used to register 
8. Then you can check whether if updating an employee's role is working 
9. The last one check that deleted employee's is removed from the displayed list of employees

## Relevant Pivotal tracker
[Registering drivers && operators](https://trello.com/c/woM53at3/36-register-drivers-operators)
